test/server/cards/characters/05/05041-wildlingbandit.spec.js

```diff
@@ -0,0 +1,45 @@
+/* global describe, it, expect, beforeEach, integration */
+/* eslint camelcase: 0, no-invalid-this: 0 */
+
+describe('Wildling Bandit', function() {
+    integration(function() {
+        beforeEach(function() {
+            const deck1 = this.buildDeck('lannister', [
+                'Sneak Attack',
+                'Wildling Bandit'
+            ]);
+            const deck2 = this.buildDeck('lannister', [
+                'Time of Plenty'
+            ]);
+            this.player1.selectDeck(deck1);
+            this.player2.selectDeck(deck2);
+            this.startGame();
+            this.keepStartingHands();
+
+            this.bandit = this.player1.findCardByName('Wildling Bandit');
+            this.player1.clickCard(this.bandit);
+            this.completeSetup();
+
+            this.player1.selectPlot('Sneak Attack');
+            this.player2.selectPlot('Time of Plenty');
+            this.selectFirstPlayer(this.player1);
+
+            this.completeMarshalPhase();
+        });
+
+        describe('when attacking a more wealthy opponent', function() {
+            beforeEach(function() {
+                this.skipActionWindow();
+
+                this.player1.clickPrompt('Military');
+                this.player1.clickCard(this.bandit);
```

Comment:
Thanks for doing this, but your test is not quite correct. Attackers aren't declared until you've selected them AND have clicked 'Done'. You need a `this.player1.clickPrompt('Done')` here to declare the bandit as attacking, after which the test passes. Good to have the test, but it doesn't reveal a problem with the card or effects engine.